java/vector/src/test/java/org/apache/arrow/vector/TestBitVectorHelper.java

```diff
@@ -22,50 +22,50 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 
 import org.apache.arrow.memory.ArrowBuf;
-import org.apache.arrow.memory.ReferenceManager;
+import org.apache.arrow.memory.BufferAllocator;
 import org.apache.arrow.memory.RootAllocator;
 import org.junit.Test;
 
-import io.netty.buffer.PooledByteBufAllocatorL;
 import io.netty.util.internal.PlatformDependent;
 
 public class TestBitVectorHelper {
   @Test
   public void testGetNullCount() throws Exception {
+    BufferAllocator root = new RootAllocator();
```

Comment:
I do not find the code for closing the allocator

Reply:
It would be nice to implement the code by try-with-resource

Reply:
thanks, agreed. Fixed!